.idea/runConfigurations.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess this file is not necessary, right?

Reply:
I just checked and this file is not in my latest commit -- no idea why it's still in here.